### PR TITLE
Documentation: Improve wording and formatting in readme

### PR DESCRIPTION
**Pull request by glensc**
I wanted to document the Python versions we support and ended up doing some other cosmetics as well.

Version | Released | Security Support | Release
-- | -- | -- | --
3.9 | 5 months ago(05 Oct 2020) | Ends in 4 years and 6 months(05 Oct 2025) | 3.9.2
3.8 | 1 year and 5 months ago(14 Oct 2019) | Ends in 3 years and 7 months(14 Oct 2024) | 3.8.8
3.7 | 2 years and 8 months ago(27 Jun 2018) | Ends in 2 years and 3 months(27 Jun 2023) | 3.7.10
3.6 | 4 years ago(23 Dec 2016) | Ends in 9 months(23 Dec 2021) | 3.6.13
3.5 | 5 years ago(30 Sep 2015) | Ended 6 months ago(13 Sep 2020) | NA
3.4 | 7 years ago(16 Mar 2014) | Ended 2 years ago(18 Mar 2019) | NA
3.3 | 8 years ago(29 Sep 2012) | Ended 3 years and 5 months ago(29 Sep 2017) | NA
2.7 | 10 years ago(03 Jul 2010) | Ended 1 year and 2 months ago(01 Jan 2020) | NA

ref:
- https://endoflife.date/python

### CODE REVIEW

**Comment by glensc** [comment]
I used py3.6 as a minimum, as I saw some comment about such version:
- https://github.com/Taxel/PlexTraktSync/pull/135

and it also happens to be a version with active support.

File: README.md [hunk 1]
```diff
@@ -21,7 +18,12 @@ the documentation and comments are lacking at best.
 
 ## Setup
 
-To setup this on your own machine, first clone or download this repo.
+The script is known to work with Python 3.6-3.9 versions.
+This matches [Python EOL][python-eol] schedule:
+
+[python-eol]: https://endoflife.date/python
```
Comment (twolaw) [inline README.md:24]: This info is not needed in the readme file. Keep it simple.
Line 21 is ok.

File: README.md [hunk 2]
```diff
@@ -36,11 +38,11 @@ pipenv run python main.py
...
-
-To connect to Trakt you need to create a new API app: Visit
-`https://trakt.tv/oauth/applications/new`, give it a meaningful name and enter
-`urn:ietf:wg:oauth:2.0:oob`as the redirect url. You can leave Javascript
-origins and the checkboxes blank.
+To connect to Trakt you need to create a new API app:
+1. Visit https://trakt.tv/oauth/applications/new
+1. give it a meaningful name
+1. enter `urn:ietf:wg:oauth:2.0:oob`as the redirect url.
+1. You can leave Javascript origins and the checkboxes blank.
```
Comment (twolaw) [inline README.md:46]: numbers 1, 2, 3, 4
first letter uppercase for all 4 steps

**Review by twolaw — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (glensc) [reply README.md:46]: no, this is markdown file. this is rendered correctly sequental numbers (see the rendered result). you don't need to manually calculate sequence. I can change to an unordered list if that bothers you.

Comment (twolaw) [reply README.md]: Ok I didn't know that. All good then.

Comment (glensc) [reply README.md]: Still need approval for a merge.

**Review by twolaw — APPROVED** [review]

**Comment by glensc** [comment]
Looks like I remembered the `# for py<3.6` comment wrong, it says below 3.6.

but probably doesn't matter, lets say Python 3.5 is unsupported as it's out of support as well.

**Pull request merged**